src/main/python/gcloud_hydra/mesos_marathon_setup.py

```diff
@@ -0,0 +1,235 @@
+# Zookeeper to keep track of the current leader of the master servers.
+# The Mesos layer, built on top of this, will provide distributed synchronization and resource handling.
+# It is responsible for managing the cluster.
+# Marathon, the cluster's distributed init system, is used to schedule tasks and hand work to the slave servers.
+
+import os
+import argparse
+import setup_helpers
+import ConfigParser
+from shell_command import shell_call
+import ntpath
+from fabric.api import *
+
+parser = argparse.ArgumentParser(description='Mesos Marathon setup script')
+parser.add_argument('--config_file', '-f', type=str,
+                    default=os.getcwd() + "/setup_config.ini", help='Setup configuration file')
+parser.add_argument('--deployment_id', '-i', type=str,
+                    help='Each cluster deployment needs to have a unique identifier. '
+                         'This will help in identifying various deployments.', required=True)
+
+parser.add_argument('--local_work_dir', '-l', type=str, default=os.environ['HOME'],
+                                        help='Script will copy all downloaded/output files in this directory')
+parser.add_argument('--ssh_key_file', '-k', type=str, default=os.environ['HOME'] + "/.ssh/id_rsa.pub",
```

Comment:
I would display what's going to happen if you don't specify one (e.g. 'default value is ~/.ssh/id_rsa')